src/main/java/org/folio/marc4ld/util/Constants.java

```diff
@@ -8,6 +8,7 @@ public class Constants {
   public static final String FIELD_UUID = "999";
   public static final char SUBFIELD_INVENTORY_ID = 'i';
   public static final char SUBFIELD_SRS_ID = 's';
+  public static final char EMPTY = ' ';
```

Comment:
I think 'SPACE' was a better name for this constant. Let us fix this in another PR.